testsuite/features/support/constants.rb

```diff
@@ -1476,11 +1502,13 @@
   'sle-manager-tools-for-micro5-pool-x86_64-5.3' => 60,
   'sle-manager-tools-for-micro5-pool-x86_64-5.4' => 60,
   'sle-manager-tools-for-micro5-pool-x86_64-5.5' => 60,
+  'sle-manager-tools-for-micro6-pool-x86_64-6.0' => 60,
   'sle-manager-tools-for-micro5-updates-x86_64-5.1' => 60,
   'sle-manager-tools-for-micro5-updates-x86_64-5.2' => 60,
   'sle-manager-tools-for-micro5-updates-x86_64-5.3' => 60,
   'sle-manager-tools-for-micro5-updates-x86_64-5.4' => 60,
   'sle-manager-tools-for-micro5-updates-x86_64-5.5' => 60,
+  'sle-manager-tools-for-micro6-updates-x86_64-6.0' => 60,
```

Comment:
The updates channel is not available as of now according to SCC, so this value still needs to be commented out.

Reply:
thanks again, now commented